### PR TITLE
feat:add list component

### PR DESCRIPTION
**Pull request by shubhamkumar648**
types of lis
1.normal list (space)
2. stack list
list1
list2

### CODE REVIEW

File: component/list.html [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/main.css" />
+    <title>GK UI</title>
+    <script
+      src="https://kit.fontawesome.com/a4417b6562.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
```
Comment (shuntzhere) [inline component/list.html]: Put the script on the bottom of the html file.

File: component/list.html [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+          </button>
+
+          <button class="button">
+            <a href="#" class="link__nostyle">Installment</a>
+          </button>
+
+          <button class="button">
+            <a href="../pages/documentation.html" class="link__nostyle"
+              >Documnetation
+            </a>
```
Comment (shuntzhere) [inline component/list.html]: There's a typo here.

File: component/list.html [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+            <span><a class="active link__nostyle" href="../component/rating.html">Rating</a></span>
+        </div>
+      </aside>
+
+      <main class="main__container p-2">
+        <h3>Lists</h3>
+        <p class="ls-s lh-l">
+          List component to display data item in ordered or unodered form
+        </p>
+        <p class="lh-s mb-4">Example: Simplelist| stacklist</p>
```
Comment (shuntzhere) [inline component/list.html]: Check for typo's here.

File: component/list.html [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+        <h6 class="lh-l">Simple List:</h6>
+        <p class="lh-m">It contain orderd or unordered</p>
+        <div class="list_main__container">
+          <div class="normal_list_container p-1">
```
Comment (shuntzhere) [inline component/list.html]: Typo here as well.

File: component/list.html [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </code>
+        </div>
+
+        <h6 class="lh-l">Stack list:</h6>
+        <p class="lh-m">
+          Stacked lists are usefull to display list of users with name, avatar,
+          and status
```
Comment (shuntzhere) [inline component/list.html:110]: Typo

File: styles/list.css [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    border: 1px solid var(--primary-color);
+}
+
+.normal-list li{
+    padding: 1rem;
+    margin-bottom: 1rem;
+    background-color: var(--grey);
+    border: hidden hidden 1px solid var(--primary-color) ;
+    text-align: center;
+}
```
Comment (shuntzhere) [inline styles/list.css]: What is hidden hidden doing here? You might want to check this out.

**Review by shuntzhere — CHANGES_REQUESTED** [review]
Check for the typos and also suggested some extra codes to remove.

File: styles/list.css (hunk 6 above)
Comment (shubhamkumar648) [reply styles/list.css]: i actually want to show only bottom border line, but here it wont work here , i will fix it

**Review by therajatg — APPROVED** [review]
see review changes suggested by shuntzhere. Other than those changes it looks fine.

**Review by raman1999 — APPROVED** [review]
everything looks good now , you can merge it

**Review by shuntzhere — APPROVED** [review]
looks good to merge now.

**Pull request merged**